bin/ingest_submission.py

```diff
@@ -34,8 +34,6 @@ def main():
                           help='The aggregation type (case insensitive).')
     argparse.add_argument('--vep_version', required=False, type=int,
                           help='VEP version to use for annotation. Only needed if running variant load.')
-    argparse.add_argument('--vep_cache_version', required=False, type=int,
-                          help='VEP cache version to use for annotation. Only needed if running variant load.')
```

Comment:
Unfortunately I'm not sure we can get rid of the manually specified `vep_cache_version` as I think we need the version to match whatever we currently have in the variant warehouse for a given species, even if there's a newer version available (maybe @sundarvenkata-EBI can confirm).

I imagine for now we just treat `vep_version` and `vep_cache_version` as optional, if they're passed then we assume the user has checked the versions and we go ahead with annotation (without checking FTP).  If they're _not_ passed, we check FTP and continue with or without annotation as you've done here.

I think this is compatible with the current workflow but also lets us evolve (soon, hopefully) to a version where we don't have to get VEP versions from the user at all... and maybe can always be running the most up-to-date version of VEP available...

Reply:
I agree with @apriltuesday's suggestion that it is better to use the current cache version (if available in annotationMetadata_2_0) for a given species so that we don't make annotations with different caches (might become a maintenance nightmare later) for the same species. But that current version can be detected from the database (as mentioned above) and does not need to come from the command line.   So the order of preference would be:

1) Cache version from the variant warehouse (if already available from annotationMetadata_2_0).
2) Latest cache version from Ensembl as @nitin-ebi has done here.

Reply:
updated the logic - 
1. getting vep_cache_version form annotationsMetadata_2_0 collection from the concerned DB (in case of multiple versions, taking the max value i.e. the latest one)
2. If there is no vep_cache_version available in DB, trying to get it from ensembl
If not found in both the places, None will be returned and annotations.skip will be set to true.

Reply:
This is great, as discussed we can also get the `vep_version` from the metadata in step 1, and use the same `vep_version` as `vep_cache_version` in step 2.  It will still not work for new species but that will come in a separate ticket.

Reply:
added logic for getting vep version as well